src/main/java/org/takes/http/MainRemote.java

```diff
@@ -83,7 +83,9 @@ public MainRemote(final Class<?> type, final String... passed) {
      */
     public void exec(final MainRemote.Script script) throws Exception {
         final File file = File.createTempFile("takes-", ".txt");
-        file.delete();
+        if (!file.delete()) {
+            file.deleteOnExit();
```

Comment:
@yegor256 because I don't want to change the current method flow, for now it assumes that even if we cannot delete the file, it is not important for the rest of the method. But If you prefer an Exception, I can throw it